airflow/contrib/operators/dataproc_operator.py

```diff
@@ -669,6 +669,11 @@ class DataProcPigOperator(BaseOperator):
     :type delegate_to: string
     :param region: The specified region where the dataproc cluster is created.
     :type region: string
+    :var dataproc_job_id: The actual "jobId" as submitted to the Dataproc API.
```

Comment:
Why do we need to add the variables to doc string?

Reply:
Just trying to document the existence of the variable, as the whole point of this change is to provide access to the _real_ job ID submitted to Dataproc by, for instance, a downstream task via xcom.  I'll add the code example for my specific use case in a comment below...

Reply:
And, by all means, if there is a better place to put this, let me know.  I just figured I'd use some standard means of documenting the instance variable, and one that is supported by Sphinx...